app.js

```diff
@@ -1,11 +1,15 @@
 // IMPORT PACKAGES
```

Comment:
All the requieres  and imports by good practice should be at the top of the file, in this way we can know which package or files are imported to our file 👍
```suggestion
// IMPORT PACKAGES
const projects = require('./data/projects.json')
const articles = require('./data/articles.json')
```